File: src/main/java/org/pixeltime/enchantmentsenhance/commands/ReloadCommand.java
package org.pixeltime.enchantmentsenhance.commands;

import org.bukkit.entity.Player;
import org.pixeltime.enchantmentsenhance.manager.DataManager;
import org.pixeltime.enchantmentsenhance.manager.SettingsManager;
import org.pixeltime.enchantmentsenhance.util.Util;

public class ReloadCommand extends SubCommand {

    @Override
    public void onCommand(Player player, String[] args) {
        SettingsManager.reloadConfig();
        SettingsManager.reloadData();
        SettingsManager.reloadLang();
        new DataManager();
        Util.sendMessage(SettingsManager.lang.getString("Config.reload"),
                player);
    }


    @Override
    public String name() {
        return "reload";
    }


    @Override
    public String info() {
        return "\n&6/enhance reload &7- " + SettingsManager.lang.getString(
                "Help.reload");
    }


    @Override
    public String[] aliases() {
        return new String[]{"reload", "rel", "chongzai", "cz"};
    }


    @Override
    public String getPermission() {
        return "Enchantmentsenhance.reload";
    }

}

File: modules/EnchantmentsEnhance-Plugin/src/main/kotlin/org/pixeltime/enchantmentsenhance/command/player/ReloadCommand.kt
package org.pixeltime.enchantmentsenhance.command.player

import org.bukkit.entity.Player
import org.pixeltime.enchantmentsenhance.command.SubCommand
import org.pixeltime.enchantmentsenhance.command.console.ReloadConsoleCommand

class ReloadCommand : SubCommand() {


    override val permission: String
        get() = "Enchantmentsenhance.reload"

    override fun onCommand(player: Player, args: Array<String>) {
        ReloadConsoleCommand().onCommand(player, args)
    }


    override fun name(): String {
        return "reload"
    }


    override fun usage(): String {
        return "/enhance reload"
    }


    override fun aliases(): Array<String> {
        return arrayOf("reload", "rel", "chongzai", "cz")
    }

}
